app/views/transactions/index.html.erb

```diff
@@ -1,17 +1,18 @@
 <p style="color: green"><%= notice %></p>
 
-<h1>Transactions</h1>
-<div class="d-flex justify-content-between px-4" >
-<%= link_to "Back", categories_path, class:"btn btn-sucess mb-3",style:"background-color: #cc6600;" %>
+<h4>Transactions</h4>
+<div class="d-flex justify-content-between px-4 fs-3" >
+<%= link_to "Back", categories_path, class:"btn btn-sucess mb-3 bg-primary text-white fs-4"%>
 <%= "Total amount:  #{@total_amount}" %>
 </div>
 <div id="transactions">
   <% @transactions.each do |transaction| %>
-      <%= render transaction %>
-      <p>
-        <%= link_to "Show this transaction", transaction %>
-      </p>
+    <p >
+    <%= link_to transaction_path(transaction) do %>
+      <%= render partial: "transaction", locals: { transaction: transaction } %>
+    <% end %>
+  </p>
```

Comment:
- [ ] As mentioned by the previous reviewer, as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Design-,You%20should%20follow%20these%20design%20guidelines%2C%20including%3A,-Colors.), Please fix the categories index page style by applying the correct color `#60B525` for the button and make the position of it and the bottom of the page.
- [ ] Kindly add the correct position for the back button to be in the top left on navbar and make it as back arrow `<` to match the given design.

| Given Design  | Given Design  | Your Design |
| ------------- | ------------- | ------------- |
| ![image](https://github.com/V0pe/Payday/assets/68796232/872fc2fb-4663-406d-8aaf-5e2dbdc07e6b) | ![image](https://github.com/V0pe/Payday/assets/68796232/7f9b1687-68bc-44ea-ae57-f80c4d5a9377)   | ![image](https://github.com/V0pe/Payday/assets/68796232/382b6441-9117-4441-a1a6-2b3280d5cf74) |

Reply:
@Omar-Muhamad Thank you. I have fixed it 

![image](https://github.com/V0pe/Payday/assets/61356487/a10f1d5f-23ab-4c66-94fc-6fa9277a5e47)
![image](https://github.com/V0pe/Payday/assets/61356487/0e396b1f-2213-47b5-a689-9a79236f16f8)